content/docs/user-guide/assets/pipeline/metadata.md

```diff
@@ -0,0 +1,85 @@
+
+There is only one requirement when **moving/renaming an asset: the metadata file must be moved/renamed as well**.
+
+Folders can be moved/renamed without having to touch metadata files, but note that all files within a folder must have a metadata file to ensure existing references do not break.
+
+Examples:
+
+* If blue.png is renamed to red.png, the metadata file blue.png.meta must be renamed to red.png.meta.
+* If textures/hat.png is moved to character/hat.png, the metadata file must be moved to character/hat.png.meta.
+
+
+If using source control, metadata files **must** be checked in along with the source asset and updated with any move/rename changes.
+
+There is one exception to the above, which is Intermediate Asset metadata files.  These metadata files follow the same rules as Intermediate Assets: they should not be modified or checked into source control.
+
+## How to Enable
+To enable a file type, create a .setreg file with the following settings:
+```json
+{
+    "O3DE":
+    {
+        "AssetProcessor":
+        {
+            "Settings":
+            {
+                "Metadata":
+                {
+                    "EnabledTypes": [
+                    ]
+                }
+            }
+        }
+    }
+}
+```
+EnabledTypes is simply an array of strings of file extension to enable.  For example, `".txt", ".png"`
+
+It is highly recommened that this setreg file be placed in a central location which is shared with other users of the project to ensure everyone is generating the same metadata files.  For a game project, it should be placed in the project directory; for gem development, it should be placed in the gem directory.  Keep in mind this is an Asset Processor wide setting, it will apply to all files the AP processes, not just the ones in the project/gem where the setting exists.
+
+Once a metadata file is generated for an asset and the asset is relocated or any references are saved using the newly generated UUID, the metadata file must be kept alongside the asset.  Not doing so could break references to the file.
+
+Note that AP will continue to use existing metadata files for assets which have them regardless of the above settings.
+
+## Renaming while AP is running
+By default, renaming assets manually while Asset Processor is running can prove difficult as AP will try to immediately create a new metadata file for the renamed asset.  To prevent this, AP can be configured to wait for a specified time before it attempts to process a "new" asset, which avoids the automatic creation of an unwanted, new metadata file.  Note that this delay will affect all newly created assets for metadata-enabled types, which is why this is off by default.  The longer the delay time, the longer it will take AP to start processing any newly created metadata-enabled file type.  In a .setreg file, add the following setting:

```

Comment:
```suggestion
By default, renaming assets manually while Asset Processor is running can prove difficult as AP will try to immediately create a new metadata file for the renamed asset.  To prevent this, AP can be configured to wait for a specified time before it attempts to process a "new" asset, which avoids the automatic creation of an unwanted, new metadata file.  

> This delay will affect all newly created assets for metadata-enabled types and is off by default.  The longer the delay time, the longer it will take AP to start processing any newly created metadata-enabled file type.

In a .setreg file, add the following setting:

```